src/commands/functions-env-clone.ts

```diff
@@ -0,0 +1,46 @@
+import * as clc from "cli-color";
+
+import { Command } from "../command";
+import { requirePermissions } from "../requirePermissions";
+import * as fenv from "../functions/env";
+import * as getProjectId from "../getProjectId";
+import * as utils from "../utils";
+
+export default new Command("functions:env:clone")
+  .description("clone environment variables from another project")
+  .option("--from <projectId>", "the project from which to clone environment variables")
```

Comment:
I'm sure this was discussed in the API proposal, but this isn't really optional - why is it an option?

Reply:
Don't see anyone calling this out - I think the intent here to mirror `functions:config:clone` command. I tried looking up design doc by @inlined from 2016(?) to see if anyone called this out, but no one did!

I do like that --from makes it explicit what the argument is about. At the end of the day `--from` flag isn't optional here either.

Reply:
Seems reasonable to me - I agree that its good to mirror the old command